### PR TITLE
Add packaging

### PR DESCRIPTION
**Pull request by bryanwweber**
- [x] Added entry into `CHANGELOG.md`

Changes proposed in this pull request:
 - Adds conda package/automatic upload
 - Adds PyPI wheel/sdist/automatic upload

@pr-omethe-us/chemked

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,30 +33,48 @@ install:
...
+  - if [[ -z "$TRAVIS_TAG" ]]; then
+      bash <(curl -s https://codecov.io/bash);
+    fi
+
+deploy:
+  provider: pypi
+  user: bryanwweber
+  password:
+    secure: "KtGv9Suy3Yi6H2h6V8A6TwJS6/9V4cXu0z8K8oFZXZAGG2ineTESOfxkkURj410yIlGo3OACjtbSVcY967AQ4mhWXWTo5iEz4wj0DlI4/BO2MTCkElFvjm2FB1bk6u2OjjWfN6u8EN2wRyXDtvHDNOoF56RWC2lIq28vfRbR9iwg/z5d4Yy3AXIBEe6c9NAhtQIlxoH18WCuZh7zJLPScW4IYISgCJEw0yUI4Tel85gSVXcvRvmeNXg95M+ejVmQ9zU2uQ7mNRDEcNhbMMRL0dKS0+wTzwCy+UWGAVJJZopYUJ6syRqbeIJ669bwMlevqb271FkbMTiF1jOOxnSc94WPPb8V/hvWuT4tYS2pvHS1SoUvSSLubZpBzbm9UJaI7ydpb4XYvz+6FseTUubOD3BfZJ/FIX6RCoPps92/FoomKuW1p0vB+O+Ywn0AkqTOIqLbpQV5jEwFVdY8rgPMyIvJbdyOsD3QoIRRI9jtwnDMhUMcrNZMXmcZuhmBmi1lLLtz5k9dRXI1rORCr62mKfprLbrd1Qz9yTE2X2BKSMcUSV77e3CPdsvBHETikVNwdEF48ox7sevnhzEq7mVpOOSg+71RHVfTdKuuq9nxDKC4HzjscbCrOX/JZvUyh660AvKXEi/OolsNIphqzPHE4VbtY+Hah6CEJMS9PjcDqrA="
+  server: https://testpypi.python.org/pypi  # Remove after testing
```
Comment (kyleniemeyer) [inline .travis.yml]: should this be the regular PyPI server?

Comment (bryanwweber) [inline .travis.yml]: I wanted to test it first

File: setup.py [hunk 2]
```diff
@@ -1,47 +1,64 @@
...
     name='pyked',
-
-    # Versions should comply with PEP440.  For a discussion on single-sourcing
-    # the version across setup.py and the project code, see
-    # https://packaging.python.org/en/latest/single_source_version.html
     version=__version__,
-
     description='Package for manipulating Chemical Kinetics Experimental Data (ChemKED) files.',
-    long_description=readme + '\n\n' + changelog,
+    long_description=long_description,
```
Comment (kyleniemeyer) [inline setup.py]: I see `desc` above, but not `long_description`?

Comment (bryanwweber) [inline setup.py]:
```
try:
    import pypandoc
    long_description = pypandoc.convert_text(desc, 'rst', format='md')
except ImportError:
    long_description = desc
```

Comment (kyleniemeyer) [inline setup.py]: Ah, I see it now!

File: .travis.yml (hunk 1 above)
Comment (kyleniemeyer) [inline .travis.yml]: OK, sounds good!

**Pull request merged**